Organize the configuration evars and order them alphabetically (#16)
rizinorg/book: src/configuration/evars.md

The Visual mode has an eval browser that is accessible through the `Vbe` command.

## Assembly Configuration

### asm.arch: `string`

Defines the target CPU architecture used for disassembling (`pd`, `pD` commands) and code analysis (`a` command). You can find the list of possible values by looking at the result of `e asm.arch=?` or `rz-asm -L`.
It is quite simple to add new architectures for disassembling and analyzing code. There is an interface for that. For x86, it is used to attach a number of third-party disassembler engines, including GNU binutils, Udis86 and a few handmade ones.

### asm.bits: `int`

> Determines width in bits of registers for the current architecture. Supported values: 8, 16, 32, 64. Note that not all target architectures support all combinations for asm.bits.

Reviewer: I would note the supported options more indicatively.

**Supported values**:
 - 8
 - 16
 - 32
 - 64

### asm.bytes: `bool`

> A boolean value used to show or hide displaying of raw bytes of instructions.

Reviewer: ```suggestion
Show or hide displaying of raw bytes of instructions.
```

Add before and after (can be in later PR)

### asm.cpu: `string`

You can use this configuration variable to define the CPU type. For example, if you had picked the architecture as AVR, you can choose your CPU type (ATmega1281, ATmega2561, etc) using `asm.cpu`.

### asm.flags: `bool`

> If defined to "true", disassembler view will have flags column.

Reviewer: ```suggestion
When set to "true", the disassembler view will have flags column.
```

### asm.lines.call: `bool`

> If set to "true", draw lines at the left of the disassemble output (`pd`, `pD` commands) to graphically represent control flow changes (jumps and calls) that are targeted inside current block. Also, see `asm.lines.out`.

Reviewer: ```suggestion
If set to "true", draw lines at the left of the disassembly output (`pd`, `pD` commands) to graphically represent control flow changes (jumps and calls) that are targeted inside the current block. Also, see `asm.lines.out`.
```

### asm.lines.out: `bool`

> When defined as "true", the disassembly view will also draw control flow lines that go outside of the block.

Reviewer: ```suggestion
When set to "true", the disassembly view will also draw control flow lines that go outside of the block.
```

> ### asm.linestyle: `bool`

Reviewer: the name for this config is no clear and does not match the description. Probably need to improve it in different PR for Rizin

Author: Will do it on another PR after discussing and after looking into it.

A boolean value which changes the direction of control flow analysis. If set to "false", it is done from top to bottom of a block; otherwise, it goes from bottom to top. The "false" setting seems to be a better choice for improved readability and is the default one.

### asm.offset: `bool`

> Boolean value which controls the visibility of offsets for individual disassembled instructions.

Reviewer: ```suggestion
Show or hide offsets for individual disassembled instructions.
```

### asm.os: `string`

> Selects a target operating system of currently loaded binary. Usually, OS is automatically detected by `rabin -rI`. Yet, `asm.os` can be used to switch to a different syscall table employed by another OS.

Reviewer: ```suggestion
Selects a target operating system for the currently loaded binary. Usually, OS is automatically detected by Rizin (`rabin -rI`). Yet, `asm.os` can be used to switch to a different syscall table employed by another OS.
```

Author: It should also be `rzbin`, isn't it?

```
It can be useful while disassembling obscure architectures.

### asm.sub.jmp: `bool`

> A boolean value used to substitute jump, call and branch targets in disassembly.

Reviewer: ```suggestion
Substitute jump, call and branch targets in disassembly.
```

Reviewer: Substitute with what?

For example, when turned on, it'd display `jal 0x80001a40` as `jal fcn.80001a40` in the disassembly.

### asm.sub.reg: `bool`

> A boolean value used to replace register names with arguments or their associated role alias.

Reviewer: ```suggestion
Replace register names with arguments or their associated role alias.
```

For example, if you have something like this:

```
│           0x080483ea      83c404         add esp, 4
│           0x080483ed      68989a0408     push 0x8049a98
│           0x080483f7      e870060000     call sym.imp.scanf
│           0x080483fc      83c408         add esp, 8
│           0x08048404      31c0           xor eax, eax
```
This variable changes it to:
```
│           0x080483ea      83c404         add SP, 4
│           0x080483ed      68989a0408     push 0x8049a98
│           0x080483f7      e870060000     call sym.imp.scanf
│           0x080483fc      83c408         add SP, 8
│           0x08048404      31c0           xor A0, A0
```

### asm.sub.rel: `bool`

> A boolean value which substitutes pc relative expressions in disassembly. When turned on, it shows the references as string references.

Reviewer: ```suggestion
A boolean value that substitutes expressions relative to the Program Counter in the disassembly. When turned on, it shows the references as string references.
```

For example:

```
0x5563844a0181      488d3d7c0e00.  lea rdi, [rip + 0xe7c]    ; str.argv__2d_:__s
```
When turned on, this variable lets you display the above instruction as:

```
0x5563844a0181      488d3d7c0e00.  lea rdi, str.argv__2d_:__s    ; 0x5563844a1004 ; "argv[%2d]: %s\n"
```

### asm.sub.section: `bool`

> Boolean which shows offsets in disassembly prefixed with the name of the section or map.

Reviewer: ```suggestion
Prefix offsets in the disassembly with the name of the section or map.
```

That means, from something like:

```
0x000067ea      488d0def0c01.  lea rcx, [0x000174e0]
```
to the one below, when toggled on.
```
0x000067ea      488d0def0c01.  lea rcx, [fmap.LOAD1.0x000174e0]
```

### asm.sub.varonly: `bool`

> Boolean which substitutes the variable expression with the local variable name.

Reviewer: ```suggestion
Substitutes the variable expression with the local variable name.
```

│       │   0x000082b4      mvn   r3, 0
```

### asm.trace: `bool`

A boolean value that controls displaying of tracing information (sequence number and counter) at the left of each opcode. It is used to assist with programs trace analysis.

## Screen Configuration

### scr.color: `int`

> This variable specifies the mode for colorized screen output: "false" (or 0) means no colors, "true" (or 1) means 16-colors mode, 2 means 256-colors mode, 3 means 16 million-colors mode. If your favorite theme looks weird, try to bump this up.

Reviewer: I would note the supported options more indicatively.

### scr.seek: `string`

This variable accepts a full-featured expression or a pointer/flag (eg. eip). If set, rizin will set seek position to its value on startup.

### scr.scrollbar: `bool`

If you have set up any [flagzones](http://book.rada.re/basic_commands/flags.html#flag-zones) (`fz?`), this variable will let you display the scrollbar with the flagzones, in Visual mode. Set it to `1` to display the scrollbar at the right end, `2` for the top and `3` to display it at the bottom.

### scr.utf8: `bool`

> A boolen variable to show UTF-8 characters instead of ANSI.

Reviewer: ```suggestion
Show UTF-8 characters instead of ANSI.
```

│       │   0x000082b4      mvn   r3, 0
```

### asm.trace: `bool`

A boolean value that controls displaying of tracing information (sequence number and counter) at the left of each opcode. It is used to assist with programs trace analysis.

## Screen Configuration

### scr.color: `int`

This variable specifies the mode for colorized screen output.

* "false" (or 0) means no colors

> * "true" (or 1) means 16-colors mode

Reviewer: Would go for only number, or for "0 (or "false")". ...
